### PR TITLE
2024 09 27 jk qene effo

### PR DESCRIPTION
**Pull request by karljonaskarlsson**
Another qene from the Uppsala collection. This is interesting, because it has two attestations in the manuscript: once on a loose leaf and once in the hand of the main scribe (= Kolmodin?). I follow the readings of the loose leaf: https://www.alvin-portal.org/alvin/imageViewer.jsf?dsId=ATTACHMENT-0457&pid=alvin-record:280997

There is a word in the first line that I can't make out, so ideas would be appreciated!

### CODE REVIEW

File: new/LIT7078QeneEffo.xml [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                </listWit>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p><foreign xml:lang="gez">Mawaddəs</foreign>-type <foreign xml:lang="gez">qǝne</foreign>
```
Comment (eu-genia) [inline new/LIT7078QeneEffo.xml:37]: `ref type="authFile" corresp="Mawaddes"` or add to keywords?

File: new/LIT7078QeneEffo.xml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <listWit>
+                    <witness corresp="UppQ1524o"/>
```
Comment (eu-genia) [inline new/LIT7078QeneEffo.xml:27]: In this case, to make the particularity more visible, I would list two wtnesses 
```
<witness corresp="UppQ1524o">Uppsala, Q 1524o, f. 30-31</witness>
<witness corresp="UppQ1524o">Uppsala, Q 1524o, f. 43</witness>
```

**Review by nafisa-valieva — APPROVED** [review]

**Review by CarstenHoffmannMarburg — CHANGES_REQUESTED** [review]

File: new/LIT7078QeneEffo.xml [hunk 3]
```diff
@@ -0,0 +1,86 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7078QeneEffo" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">እፎ፡ አይሁድ፡ አዝማደ፡ እምከ፡ <surplus resp="JK">ተ</surplus>ደብተራ፡ መስቀል፡ ተከሉ፡ በአውደ፡ መትከፍትከ፡ ሴሎ።</title>
```
[the remark this answers is not in the document]
Comment (karljonaskarlsson) [reply new/LIT7078QeneEffo.xml:8]: Thanks! :)

File: new/LIT7078QeneEffo.xml [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+                <title xml:lang="gez" corresp="#t1" type="normalized">ʾƎffo ʾayhud ʾazmāda ʾəmməka dabtarā masqal takalu ba-ʾawda matkaftəka selo (Qǝne of the mawaddəs-type)</title>
+                <title xml:lang="en" corresp="#t1">How (could) the Jews, the relatives of your mother, the Tabernacle, plant the Cross in the courtyard of your shoulders, Selo…</title>
```
[the remark this answers is not in the document]
Comment (karljonaskarlsson) [reply new/LIT7078QeneEffo.xml:10]: I don't know if "around" makes much more sense, but I can change it :)

**Review by CarstenHoffmannMarburg — APPROVED** [review]

**Pull request merged**